src/wagtail_live/publishers.py

```diff
@@ -0,0 +1,238 @@
+"""Wagtail Live publisher classes."""
+
+import time
+from datetime import datetime
+from functools import cached_property
+
+from django.http import JsonResponse
+from django.shortcuts import get_object_or_404
+from django.urls import path
+from django.views import View
+
+from .utils import get_live_page_model, get_polling_interval, get_polling_timeout
+
+
+class PollingPublisherMixin(View):
+    """A mixin for publishers using the polling technique.
+
+    Attributes:
+        url_path (str):
+            Path of the URL used by client side to fetch new updates.
+        url_name (str):
+            Name of the URL for reversing/resolving.
+    """
+
+    url_path = "get-updates/<str:channel_id>/"
+    url_name = ""
+
+    @cached_property
+    def model(self):
+        """Retrieves the live model defined."""
+
+        return get_live_page_model()
+
+    @classmethod
+    def get_urls(cls):
+        """Retrieves the URLs client side uses to fetch updates."""
+
+        return [
+            path(cls.url_path, cls.as_view(), name=cls.url_name),
+        ]
+
+    @staticmethod
+    def get_last_update_client_from_request(request):
+        """Retrieves the timestamp of the last update received
+        in the client side.
+
+        Args:
+            request (HttpRequest): client side request
+
+        Returns:
+            (float) timestamp of the last update received in the client side.
+        """
+
+        return float(request.GET.get("last_update_ts"))
```

Comment:
I think that this can happen only if `page.latest_revision_created_at` is `None`. And if that happens, we will be in trouble sooner since we call the `timestamp` method on None object [here](https://github.com/wagtail/wagtail-live/blob/35536fb027eec58f425d0a6700979181b6c3ff5a/src/wagtail_live/models.py#L43).  So, we need to handle it from there I think. How do you suggest to handle it?

Reply:
Hmm, ok. Maybe we shouldn't worry too much and just return a 400 bad request response to the client if it doesn't include a `last_update_ts ` parameter.